cmd/cronosd/cmd/fix-unlucky-tx.go

```diff
@@ -101,14 +119,34 @@ func FixUnluckyTxCmd() *cobra.Command {
 					func(baseApp *baseapp.BaseApp) { baseApp.SetCMS(cms) },
 				)
 			}
-
 			// replay and patch a single block
-			processBlock := func(height int64) error {
+			action := "patched"
+			var f *os.File
+			if exportToFile != "" {
+				f, err = os.OpenFile(exportToFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
```

Comment:
`O_APPEND` by default may not be intuitive, I would expect it'll override the output file.
If we support stdout, user can append by:
`cronosd --export-to-file - >> /tmp/output`

Reply:
is it ok `f, err = os.Create(exportToFile)` since higher permission 0666

Reply:
I think `os.Create` is better, as long as we support stdout, the user can do append in the shell anyway.